### ISSUE TITLE
Confusing warning message when input has no value

### ISSUE THREAD

**Issue by clintoncwolfe** [inspec/inspec#4445]
# Describe the problem

As reported [here](https://github.com/inspec/inspec/issues/3802#issuecomment-526972910), if an input in the DSL does not have a value, the warning message can read:
```
WARN: Input 'cis_level' does not have a value. Use --input-file to provide a value for 'cis_level' or specify a value with attribute('cis_level', value: 'somevalue', ...)
```

Note that it uses the term `attribute(...)`, which is out of date.

**Comment by clintoncwolfe** [comment]
The correct language to use in the profile DSL is `input(...)` - though `attribute(...)` is supported for backwards compatibility. Thanks for letting us know about the messaging.

**Comment by mcascone** [comment]
I'd like to add that there might be a cousin issue here:
This command:
```powershell
inspec exec ..\inspec-ice\ -t "winrm://$ENV:USERNAME@$master.mydomain.com" --password $ENV:VRA_PASS --input=role=master sql_server=$sql
```

The input is correctly passed to the execution, however it gives this warning:
```
 WARN: Input 'sql_server' does not have a value. Use --input-file to provide a value for 'sql_server' or specify a  value with `attribute('sql_server', value: 'somevalue', ...)`.
```

**Comment by zpandey** [comment]
I am seeing the same warning messages even though the value has been provided. It actually gets the value and run the control successfully but still gives warning on the top. Is it going to be fixed soon?

[2020-02-07T16:33:50-05:00] WARN: Input 'hostname' does not have a value. Use --input-file to provide a value for 'hostname' or specify a  value with `attribute('hostname', value: 'somevalue', ...)`.
[2020-02-07T16:33:50-05:00] WARN: Input 'phase' does not have a value. Use --input-file to provide a value for 'phase' or specify a  value with `attribute('phase', value: 'somevalue', ...)`.
[2020-02-07T16:33:50-05:00] WARN: Input 'domain' does not have a value. Use --input-file to provide a value for 'domain' or specify a  value with `attribute('domain', value: 'somevalue', ...)`.
[2020-02-07T16:33:50-05:00] WARN: Input 'swind_user' does not have a value. Use --input-file to provide a value for 'swind_user' or specify a  value with `attribute('swind_user', value: 'somevalue', ...)`.
[2020-02-07T16:33:50-05:00] WARN: Input 'swind_pass' does not have a value. Use --input-file to provide a value for 'swind_pass' or specify a  value with `attribute('swind_pass', value: 'somevalue', ...)`.

Profile: Check the Existence of a Server in Solarwinds (verify-swind)
Version: 0.1.0
Target:  winrm://sa_puppetagent@http://10.33.83.46:5985/wsman:3389

  ✔  Existence of server in Solarwinds, runs on PowershellHost: Check if XXXXXXX Exists in Solarwinds
     ✔  Powershell strip is expected to eq ""
     ✔  Powershell stderr is expected to eq ""
     ✔  Powershell strip.length is expected to eq 0


Profile Summary: 1 successful control, 0 control failures, 0 controls skipped
Test Summary: 3 successful, 0 failures, 0 skipped

**Comment by james-stocks** [comment]
Fixed in 4.18.111
